### ISSUE TITLE
How did you get the show-more-modal component to work?

### ISSUE THREAD

**Issue by Luid101** [leandrobortoli/ng-vertical-filter-bar#1]
I integrated your code into my application, but I can't get it to work for some reason. It looks like u are calling a href="#{{ model.identifier }}-show-more-modal" when the "show more" button is clicked. Is that enough?

**Comment by leandrobortoli** [comment]
It should be enough. I used the w3schools tutorial as a reference: https://www.w3schools.com/bootstrap/bootstrap_ref_js_modal.asp. It has a note saying `For <a> elements, omit data-target, and use href="#modalID" instead`.

I developed this project with bootstrap 4.1, but I have just tested with bootstrap 4.5 and it worked just fine. Try adding data-target="#{{ model.identifier }}-show-more-modal as well to see if that works.
